docs/resources/object_bucket.md

```diff
@@ -21,6 +21,85 @@ resource "scaleway_object_bucket" "some_bucket" {
 }
 ```
 
+### Using object lifecycle
+
+```hcl
+resource "scaleway_object_bucket" "main"{
+  name = "mybuckectid"
+  region = "fr-par"
+  acl = "private"
+  
+  lifecycle_rule {
+      id      = "id1"
+      prefix  = "path1/"
+      enabled = true
+  
+      expiration {
+        days = 365
+      }
+  
+      transition {
+        days          = 120
+        storage_class = "GLACIER"
+      }
+  }
+  
+  lifecycle_rule {
```

Comment:
For all those lifecycle_rule, could you add a description in English that explains what they do?